mappings/clacso.py

```diff
@@ -67,7 +69,7 @@ def _get_identifiers(self, record, namespaces):
     def _get_dates(self, record, namespaces):
         years_issued = [date.split('-')[0] if '-' in date else date for date in record.xpath('./dc:date/text()', namespaces=namespaces)]
         
-        return [f'{min(years_issued)}|issued']
+        return [f'{min(years_issued)}|publisher_date']
```

Comment:
should this be publication_date or publisher_date?

Reply:
It should be publication_date.